components/Dialogs/src/MDCAlertController.h

```diff
@@ -152,13 +187,28 @@ typedef void (^MDCActionHandler)(MDCAlertAction *_Nonnull action);
 @interface MDCAlertAction : NSObject <NSCopying, UIAccessibilityIdentification>
 
 /**
- Action alerts control the buttons that will be displayed on the bottom of an alert controller.
+ A convenience method for adding actions that will be rendered as low emphasis buttons at the
+ bottom of an alert controller.
+
+ @param title The title of the button shown on the alert dialog.
+ @param handler A block to execute when the user selects the action.
+ @return An initialized MDCActionAlert object.
+ */
++ (nonnull instancetype)actionWithTitle:(nonnull NSString *)title
+                                handler:(__nullable MDCActionHandler)handler;
+
+/**
+ An action that renders at the bottom of an alert controller as a button of the given emphasis.
 
  @param title The title of the button shown on the alert dialog.
+ @param emphasis The emphasis of the button that will be rendered in the alert dialog.
+        Unthemed actions will render all emphases as text. Apply themers to the alert
+        to achieve different appearance for different emphases.
  @param handler A block to execute when the user selects the action.
  @return An initialized MDCActionAlert object.
  */
 + (nonnull instancetype)actionWithTitle:(nonnull NSString *)title
+                               emphasis:(MDCActionEmphasis)emphasis
                                 handler:(__nullable MDCActionHandler)handler;
```

Comment:
Consider annotating as the designated initializer.

Reply:
Nevermind - this is a static method and can't be marked as a designated initializer.